setup.cfg

```diff
@@ -52,6 +52,8 @@ scripts =
     bin/pypeit_cp_spec1d
 
 [options.extras_require]
+scikit-image =
```

Comment:
Do we still need this? I'm not sure anyone is using it?

Reply:
removing and we'll see if things break